docs/3.-Sprints-&-Teams/Team-Structure.md

```diff
@@ -1,3 +1,37 @@
-Ahh! This one hasn't been written yet!
+---
+title: Team Structure
+authors: James Dingle
+reviewed: 
+reviewer:
+next-review: 01-04-2022
+---
+
+# Team Structure
+
+The Platform Development department is split up into a number of self-organised, cross-functional delivery teams, plus an enablement team. Delivery teams usually have a lead engineer, three software engineers, a test engineer and product manager.
```

Comment:
In the name of consistency, perhaps we could continue the nomenclature "Product Delivery Teams" "https://github.com/amdigital-co-uk/amdigital-co-uk.github.io.git". It's a bit complicated I guess because that doesn't include the tech facilitation team!

Reply:
Thanks for the feedback, I've updated to reflect your comments. 😄 Consistency is important!  

Is there another term we could use that would encompass both? Or is 'Product Delivery Team' good enough?